presentation/matrix_view.py

```diff
@@ -4,72 +4,124 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_table
+import numpy as np
 from dash.dependencies import Input, Output
 
 from domain.config import Config
 from domain.model import MeshResults
 from presentation.matrix import Matrix
 
+RED = "rgb(255,0,0)"
+ORANGE = "rgb(255,165,0)"
+GREEN = "rgb(0,255,0)"
 
-def make_mesh_test_matrix_layout(mesh: MeshResults, config: Config) -> dash_table.DataTable:
-    column_zero_header = {"name": "", "id": "agent_alias"}  # left-top table cell
-    columns = [column_zero_header] + [{"name": row.alias, "id": row.alias} for row in mesh.rows]
+
+def make_mesh_test_matrix_layout(mesh: MeshResults, config: Config) -> dcc.Graph:
     matrix = Matrix(mesh)
-    data = make_data(mesh, matrix)
-    styles = make_colors(mesh, config.latency_deteriorated_ms, config.latency_failed_ms)
-    return dash_table.DataTable(
-        id="table",
-        columns=columns,
-        style_cell={"textAlign": "center"},
-        data=data,
-        style_data_conditional=styles,
+    data = make_data(mesh, matrix, config.latency_deteriorated_ms, config.latency_failed_ms)
+    annotations = make_annotations(mesh, matrix)
+    layout = dict(
+        margin=dict(l=150, b=50, t=100, r=50),
+        modebar={"orientation": "v"},
+        annotations=annotations,
+        xaxis=dict(side="top", ticks="", scaleanchor="y"),
+        yaxis=dict(side="left", ticks=""),
+        hovermode="closest",
+        showlegend=False,
     )
 
+    return dcc.Graph(figure={"data": data, "layout": layout}, style={"width": 750, "height": 750})
+
 
-def make_data(mesh: MeshResults, matrix: Matrix) -> List[Dict]:
-    data = []
-    for row in mesh.rows:
-        row_data = {"agent_alias": row.alias}
+def make_data(mesh: MeshResults, matrix: Matrix, latency_deteriorated_ms: int, latency_failed_ms: int) -> List[Dict]:
+    x = matrix.agents
+    y = [i for i in reversed(matrix.agents)]
+    z = []
+
+    for row in reversed(mesh.rows):
+        z_col = []
         for col in row.columns:
-            latency_ms = matrix.cells[row.alias][col.alias].latency_microsec.value / 1000  # scale to milliseconds
-            row_data[col.alias] = f"{latency_ms} ms"
-        data.append(row_data)
+            color = get_color(
+                matrix.cells[row.alias][col.alias].latency_microsec.value, latency_deteriorated_ms, latency_failed_ms
+            )
+            z_col.append(color)
+        z.append(z_col)
+    for i in range(len(mesh.rows)):
```

Comment:
Please add a comment or even better extract those two lines to a function called like `unset_diagonal_color()`

Reply:
I added it, I think it was deleted during resolving conflicts by accident